app/components/layout/page-header.tsx

```diff
@@ -1,39 +1,58 @@
-import { CheckBox, Sports } from "@mui/icons-material";
-import { Grid, Typography, Divider, Box } from "@mui/material";
 import React from "react";
+import Link from "next/link";
+import { AppBar, Box, Grid, Toolbar, Typography } from "../providers/mui-server-components";
+import { MaetIcon } from "../icons";
+import { SearchBar } from "../user-input";
+import { AuthButton } from "../auth-button";
 
 export interface PageHeaderProps {
 	title: string;
 }
 
 export /**
- * Reusable page header
+ * Reusable page header/toolbar
  *
  * @param {*} {
  *		title,
  *	}
  *  @return {*}
  *
- */ const PageHeader = (props: PageHeaderProps) => (
-	<Box
-		justifyContent="center"
-		alignItems={"center"}
-		sx={{
-			display: "flex",
-			textAlign: "center",
-			backgroundColor: "#4f46e5",
-			border: 1,
-			borderRadius: 1,
-			borderColor: "#4f46e5",
-		}}
-	>
-		<Typography
-			fontFamily={["Nunito", "sans-serif"]}
-			color="white"
-			variant="h5"
-			fontWeight="800"
-		>
-			{props.title}
-		</Typography>
+ */ const PageHeader = () => (
```

Comment:
A little confused, code looks good, but why do we have two pageHeader components?